monai/transforms/spatial/array.py

```diff
@@ -483,13 +486,23 @@ def __init__(
             recompute_affine: whether to recompute affine based on the output shape. The affine computed
                 analytically does not reflect the potential quantization errors in terms of the output shape.
                 Set this flag to True to recompute the output affine based on the actual pixdim. Default to ``False``.
+            min_pixdim: minimally allowed input spacing. If provided, input image with a larger spacing than this value
+                will be kept the same (not be resampled to `pixdim`) by this transform, default to `None`.
+            max_pixdim: maximally allowed input spacing. If provided, input image with a smaller spacing than this value
+                will be kept the same (not be resampled to `pixdim`) by this transform, default to `None`.
```

Comment:
I feel the doc-string for the 2 args are opposite?

Thanks.

Reply:
the current is that according to the input image's `pixdim`,  if `min_pixdim < pixdim < max_pixdim` is true, then it will skip the resampling...

Reply:
OK, so I think maybe the doc-string is not very clear: "If provided, input image with a larger spacing than this value will be kept the **same**". I would suggest to change "same" to "original".
What do you think?
CC @ericspod 

Thanks.